test.md

```diff
@@ -47,19 +47,48 @@ We allow 2 kinds of actions:
 
 ```k
     syntax Auxil  ::= Action
-    syntax Action ::= "(" "invoke"         String        ")"
-                    | "(" "invoke"         String Instrs ")"
-                    | "(" "get"            String        ")"
-                    | "(" "get" Identifier String        ")"
- // --------------------------------------------------------
-    rule <k> ( invoke ENAME:String IS:Instrs ) => IS ~> ( invoke ENAME ) ... </k>
-    rule <k> ( invoke ENAME:String )           => ( call TFIDX )         ... </k>
+    syntax Action ::= "(" "invoke" OptionalId String Instrs ")"
+                    |     "invoke" Int        String
+                    | "(" "get"    OptionalId String        ")"
+                    |     "get"    Int        String
+ // ------------------------------------------------
+    rule <k> ( invoke OID:OptionalId ENAME:String IS:Instrs ) => IS ~> ( invoke OID ENAME .Instrs ) ... </k>
+      requires IS =/=K .Instrs
+
+    rule <k> ( invoke ENAME:String .Instrs ) => invoke CUR ENAME ... </k>
```

Comment:
Let's use the actual klabel, lo LHS looks like `( invoke .Identifier ENAME:String .Instrs )`

Reply:
We found that this doesn't work.